ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/WhitelistDataAccess.java

```diff
@@ -58,6 +58,7 @@
 {
     private static final Logger LOG = LoggerFactory.getLogger(WhitelistDataAccess.class);
 
+    private static final int MAX_WAIT_TIME_FOR_SCHEMA_ALIGNMENT_IN_SECONDS = 120;
```

Comment:
Many wait times in Cassandra are configurable. Would it make sense to have this configurable from a system property (with a default of 120)?